scripts/i18n/utils/parser.ts

```diff
@@ -0,0 +1,80 @@
+import fs from 'node:fs/promises';
```

Comment:
is there a preexisting parser of .po in the codebase?

Reply:
we have https://www.npmjs.com/package/gettext-parser in dependencies, i can use it instead of our parser